Sources/SpeziLLMLocal/Helpers/LLMModel+numParameters.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LLMModel+numParameters.swift
+//  SpeziLLM
+//
+//  Created by Leon Nissen on 10/14/24.
+//
```

Comment:
Please ensure that all headers follow our shared setup across all projects; you might also investigate all the REUSE errors.